### PR TITLE
Improve description of the HTTP JSON API in the Ledger API docs

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
changelog_begin
changelog_end

Fixes https://github.com/digital-asset/daml/issues/13029

Summary of changes:
- removed the reference to the fact that one needs to use _another_ language to access the Ledger API (Daml Script is still Daml)
- moved ways to access the Ledger API into its own paragraph:
  - suggest first to use the Java bindings
  - be more explicit about how gRPC enters the picture
  - be more specific about how the HTTP JSON API enters the picture

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by stefanobaghino-da** [comment]
@carrielaben-da Thanks for your comments on the draft I first created [here](https://github.com/digital-asset/daml/issues/13029). Since the team seemed to be onboard with the changes, I improved it a bit and further addressed the comments you left [as part of this commit](https://github.com/digital-asset/daml/pull/13271/commits/1195325fcf9797332ee430f1bc020e68163d3ede). One of the typos you found was already solved as part of my first pass.

File: docs/source/app-dev/ledger-api.rst [hunk 1]
```diff
@@ -17,14 +17,14 @@ The Ledger API
    bindings-x-lang/index
 
 
-To write an application around a Daml ledger, you'll need to interact with the **Ledger API** from
-another language. Every ledger that Daml can run on exposes this same API.
+To write an application around a Daml ledger, you'll need to interact with the **Ledger API**.
```
Comment (carrielaben-da) [inline docs/source/app-dev/ledger-api.rst:20]:
```suggestion
To write an application around a Daml ledger, you will need to interact with the **Ledger API**.
```

File: docs/source/app-dev/ledger-api.rst [hunk 2]
```diff
@@ -52,6 +52,22 @@ For full information on the services see :doc:`/app-dev/services`.
 
 You may also want to read the :doc:`protobuf documentation </app-dev/grpc/proto-docs>`, which explains how each service is defined as protobuf messages.
 
+How to Access the Ledger API
+****************************
+
+You can access the Ledger API via the :doc:`Java Bindings <bindings-java/index>`.
+
+If you don't use a language that targets the JVM you can use gRPC to generate the code to access the Ledger API in
```
Comment (carrielaben-da) [inline docs/source/app-dev/ledger-api.rst:60]:
```suggestion
If you don't use a language that targets the JVM, you can use gRPC to generate the code to access the Ledger API in
```

File: docs/source/app-dev/ledger-api.rst [hunk 3]
```diff
@@ -52,6 +52,22 @@ For full information on the services see :doc:`/app-dev/services`.
...
+several supported programming languages. :doc:`Further documentation <bindings-x-lang/index>` provides a few
+pointers on how you may want to approach this.
+
+You can also use the :doc:`HTTP JSON API Service </json-api/index>` to tap into the Ledger API.
+
+At its core, it provides a simplified view of the active contract set and additional primities to query it and
```
Comment (carrielaben-da) [inline docs/source/app-dev/ledger-api.rst:66]:
```suggestion
At its core, this service provides a simplified view of the active contract set and additional primities to query it and
```

Comment (carrielaben-da) [reply docs/source/app-dev/ledger-api.rst:66]: Also, is "primities" correct here?

File: docs/source/app-dev/ledger-api.rst [hunk 4]
```diff
@@ -52,6 +52,22 @@ For full information on the services see :doc:`/app-dev/services`.
...
+exposing it using a well-defined JSON-based encoding over a conventional HTTP connection.
+
+A subset of the services mentioned above is also available as part of it.
```
Comment (carrielaben-da) [inline docs/source/app-dev/ledger-api.rst:69]: As part of what?

File: docs/source/app-dev/ledger-api.rst [hunk 5]
```diff
@@ -52,6 +52,22 @@ For full information on the services see :doc:`/app-dev/services`.
...
+If you don't use a language that targets the JVM, you can use gRPC to generate the code to access the Ledger API in
+several supported programming languages. :doc:`Further documentation <bindings-x-lang/index>` provides a few
+pointers on how you may want to approach this.
+
+You can also use the :doc:`HTTP JSON API Service </json-api/index>` to tap into the Ledger API.
+
+At its core, it provides a simplified view of the active contract set and additional primities to query it and
+exposing it using a well-defined JSON-based encoding over a conventional HTTP connection.
+
+A subset of the services mentioned above is also available as part of it.
```
Comment (stefanobaghino-da) [inline docs/source/app-dev/ledger-api.rst:69]:
```suggestion
A subset of the services mentioned above is also available as part of the HTTP JSON API.
```

File: docs/source/app-dev/ledger-api.rst (hunk 3 above)
Comment (stefanobaghino-da) [reply docs/source/app-dev/ledger-api.rst]: Addressed by https://github.com/digital-asset/daml/pull/13271/commits/eec658d36352eb6efe99ed501d80b68f1a12e050

File: docs/source/app-dev/ledger-api.rst (hunk 4 above)
Comment (stefanobaghino-da) [reply docs/source/app-dev/ledger-api.rst]: Addressed by https://github.com/digital-asset/daml/pull/13271/commits/ac92165a493ddc851efe425f173e6c31c0d8ecf7

**Review by S11001001 — APPROVED** [review]

**Pull request merged**